sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -571,12 +573,22 @@ def to_remote_storage(self) -> List[str]:
 
         table = self.to_bigquery()
 
+        parquet_file_size_mb = os.getenv("BQ_EXPORT_PARQUET_FILE_SIZE_MB")
```

Comment:
A quick check to ensure this isn't larger than 1G would be nice

Reply:
It can't be more then 1G. So perhaps it would be nice to add a check that you haven passed more than 1000

Reply:
That's exactly what I said 😁